### DISCUSSION TITLE
duda WITH-YEAR-DISTRICT

### DISCUSSION THREAD

**Discussion by rominaarancibia** [IIC2133-PUC/2024-1#106] [Tarea 1] [answered]
Hola! No me queda del todo claro lo de ordenar por año. Gracias!
<img width="1512" alt="Captura de pantalla 2024-04-22 a la(s) 9 39 25 p  m" src="https://github.com/IIC2133-PUC/2024-1/assets/127449220/eaacdecb-81a0-4542-b4bc-0ab497250270">

**Comment by sebacarrascop** [comment]
hola!, debes buscar todas las cabezas que sean del año entregado en la consulta y listarlas según el formato de output en orden de llegada, es decir que si al entregar las n cabezas te llego la con id 5 antes que la del id 3 y ambas son del año solicitado, debes entregar la cabeza 5 y luego la 3, ya que la 5 se entregó antes que la 3.

**Comment by sachondo7** [comment, marked as answer]
Ese comando tiene que retornar todas las cabezas que cumplan con ser de ese año y de ese distrito. Además, tiene que imprimirlas según el orden de llegada en caso de haber mas de una, queda claro ahí? 
Saludos!
